packages/app/src/specs/SpecFileItem.vue

```diff
@@ -9,17 +9,21 @@
       group-hocus:icon-light-indigo-600"
       :class="selected ? 'icon-dark-indigo-300 icon-light-indigo-600' : 'icon-dark-gray-800 icon-light-gray-1000'"
     />
-    <HighlightedText
-      :text="fileName"
-      :indexes="indexes.filter((idx) => idx < fileName.length)"
-      class="font-medium pl-8px whitespace-nowrap"
-      :class="selected ? 'text-white' : 'group-focus:text-indigo-300 text-gray-400 group-hover:text-indigo-300'"
-    />
-    <HighlightedText
-      :text="extension"
-      :indexes="indexes.filter((idx) => idx >= fileName.length).map(idx => idx - fileName.length)"
-      class="text-gray-700"
-    />
+    <div class="text-gray-400 truncate">
```

Comment:
Needed to apply the text color class to the outer container otherwise the ellipsis wouldn't match the text color for the spec